### DISCUSSION TITLE
Integration of proteomics data

### DISCUSSION THREAD

**Discussion by alinarekena** [SysBioChalmers/GECKO#343] [Q&A]
Hi! Wanted to ask an advice on the integration of proteomics data at STAGE 4. I'm getting an infeasible ecModel at STEP 19 after applying enzyme concentrations. The issue doesn't even allow to update protein pool and load flux data. I tried with adjusting biomass composition (protein/carb, lipid content), but it didn't help. Is there anything major different at this stage now? Because with GECKO 2 the same data worked.

**Comment by mihai-sysbio** [comment]
Hi @alinarekena! Thanks for giving GECKO3 a try, I remember talking to you about it back at the ME conference.

> an infeasible ecModel at STEP 19 after applying enzyme concentrations

Just to confirm, is this just after [this line]( https://github.com/SysBioChalmers/GECKO/blob/b512ea321b3001c1ff1c6140baccd3a3566e23d0/tutorials/full_ecModel/protocol.m#L313C23-L313C23) in STEP 19:

```
ecModel = fillEnzConcs(ecModel,protData);
```

**Comment by alinarekena** [comment]
Hi @mihai-sysbio! 
No, it occured after this line in STEP 19:
`ecModel = constrainEnzConcs(ecModel);`
